README.md

```diff
@@ -37,7 +39,8 @@ Authentic.ensure_valid(request.cookies[:token])
 
 # Manually pass in iss_whitelist
 opts = { iss_whitelist: ['https://articulate.auth0.com/'] }
-validator = Authentic::Validator.new opts
+Authentic::Validator.configure(opts)
+validator = Authentic::Validator.instance
```

Comment:
We need to update to `Authentic::Validator.new` to match recent changes

Reply:
48e528105f85fc5745fed699240d94f343590b67